bin/pin-dependency.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Istio Authors.
```

Comment:
Not sure if we can simply pull code with a Kubernetes Authors copyright and replace it with an Istio authors one? This is pervasive across the files.